sources/packages/backend/workflow/test/test-utils/factories/student-dependent.ts

```diff
@@ -93,3 +104,44 @@ export function createFakeStudentDependentNotEligible(
       };
   }
 }
+
+/**
+ * Create a student dependant who is eligible for child care costs.
+ * @param eligibility eligibility rule.
+ * @param studyStartDate study start date of the offering.
+ * @returns eligible dependant.
+ */
+export function createFakeStudentDependentEligibleForChildcareCost(
+  eligibility: DependentChildCareEligibility,
+  studyStartDate: Date | string,
+): StudentDependent {
+  switch (eligibility) {
+    case DependentChildCareEligibility.Eligible0To11YearsOld:
+      return {
+        dateOfBirth: addToDateOnlyString(studyStartDate, -11, "years"),
+        attendingPostSecondarySchool: YesNoOptions.No,
+        declaredOnTaxes: YesNoOptions.No,
+      };
+    case DependentChildCareEligibility.Eligible12YearsAndOver:
+      return {
+        dateOfBirth: addToDateOnlyString(studyStartDate, -12.1, "years"),
```

Comment:
Shouldn't `-12` be enough? As I understood it is 12 y.o. or over.

Reply:
You are right, 12 is sufficient to create the dependent in that category. I kept it as 12 above just because it also qualifies.

I can change that.